src/main/java/frc/robot/subsystems/CDSSubsystem.java

```diff
@@ -182,9 +185,21 @@ public String senseColor() {
     }
   }
 
-  /*public boolean managementEnabled() {
-    return SmartDashboard.getBoolean("Ball Management Enabled", true);
-  }*/
+  public boolean sensorsOnline() {
+    sensorStatuses = colorSensors.getProximities();
+    for (int prox : sensorStatuses) {
+      if (prox == 0) {
+        sensorsDown += 1;
```

Comment:
Should we make this a bool instead? It will not count how many sensors are down because we break the loop on the first one we find. It will however accumulate every time we call it and a sensor is down, even the same sensor as before. Might be easier to use this value if it is true/false